README.md

```diff
@@ -44,26 +44,34 @@ After that, build and run the simulator:
 
 All paramters to `run.sh` are passed to waf, i.e. the command run inside the container will be `./waf --run "simple-p2p --delay=10ms --bandwidth=10Mbps"`.
 
-### Running the endpoints
+### Building your own QUIC endpoint
 
-The [endpoint](endpoint) directory contains the base docker image for an endpoint container. The built image is available on [dockerhub](https://hub.docker.com/r/martenseemann/quic-network-simulator-endpoint).
+The [endpoint](endpoint) directory contains the base docker image for an endpoint container. 
+The pre-built image is available on [dockerhub](https://hub.docker.com/r/martenseemann/quic-network-simulator-endpoint).
 
-If you want to build the endpoint image, you can do so by running
+When building a Docker image for your own QUIC implementation, use it as the base image:
+```docker
```

Comment:
I would show 3 sections in the dockerfile first: base image, building your endpoint, running the endpoint. Then go into what each section is.